tests/unit/validate-password-test.js

```diff
@@ -0,0 +1,39 @@
+var simple = require('simple-mock')
+var test = require('tape')
+
+var rewire = require('rewire')
```

Comment:
@benwhite-deltas never used [rewire](https://www.npmjs.com/package/rewire), looks good! We use [proxyquire](https://www.npmjs.com/package/proxyquire) in other projects. Do you see a benefit of one over the other? If not I’d stick with `proxyquire` to make it simpler for contributors to jump between repositories